docs/topics/media-pipeline.rst

```diff
@@ -197,6 +197,25 @@ For self-hosting you also might feel the need not to use SSL and not to verify S
     AWS_USE_SSL = False # or True (None by default)
     AWS_VERIFY = False # or True (None by default)
 
+
+Custom Amazon S3 headers in FileStorage
+
+Steps to pass **SSEKMSKeyId** and **ServerSideEncryption** to **scrapy** FilesPipeline using **AWS S3**
+
+Configuration Steps::
+
+    1. Subclass ``scrapy.pipelines.files.S3FilesStore``, 
```

Comment:
You can use ``` :class:`~scrapy.pipelines.files.S3FilesStore` ``` instead, which links to the reference documentation, and displays `S3FilesStore` only.

Reply:
Thanks, I'm implemented it too.